deploy-edgeX.sh

```diff
@@ -44,12 +48,15 @@ run_service config-seed
 
 if [ "$SECURITY_SERVICE_NEEDED" = "true" ]; then
 
-	run_service vault
-
-	sleep 20s
+	run_service security-secrets-setup
 
 	run_service vault-worker
 
+	run_service vault
```

Comment:
Shouldn't we start vault first before vault-worker to run?

Reply:
yes. resolved.